### PR TITLE
Brand: import GEL_GROUP_3_SCREEN_WIDTH_MAX correctly

### LINKED ISSUE #201: GEL_GROUP_2_SCREEN_WIDTH_MIN must come from breakpoints
**Describe the bug**
psammead-brand imports `GEL_GROUP_3_SCREEN_WIDTH_MAX` from `@bbc/gel-constants/spacings` - actually, this should be imported from `@bbc/gel-constants/breakpoints`.

What this results in is a malformed media query, as illustrated from the psammead-brand snapshot test: 

Source: https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-brand/src/__snapshots__/index.test.jsx.snap#L57
```
@media (min-width:) {
  .c0 {
    padding: 0 1rem;
  }
}
```

**To Reproduce**
Open https://github.com/BBC-News/psammead/blob/latest/packages/components/psammead-brand/src/__snapshots__/index.test.jsx.snap#L57

- [X] Initially labelled with ["bug"](https://github.com/BBC-News/psammead/labels/bug)

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #201

_During [integrating the psammead brand comp in simorgh](https://github.com/BBC-News/simorgh/pull/1068) it was spotted that the CSS in the snapshots had a regression which was quickly identified to be caused by an import bug, this PR fixes the bug._

In layman's terms, this PR adds padding left and right of 1rem (16px with default browser zoom) to the brand component above 1007px width.

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [x] Test engineer approval

### CODE REVIEW

**Review by bcmn — DISMISSED** [review]

**Review by radiocontrolled — DISMISSED** [review]
👍

File: packages/components/psammead-brand/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Psammead Brand Changelog
 
-| Version | Description |
-|---------|-------------|
-| 0.1.0 | [PR#105](https://github.com/BBC-News/psammead/pull/105) Create initial package, pulled in from [simorgh](https://github.com/BBC-News/simorgh). |
+| Version | Description                                                                                                                                                            |
+| ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 0.1.0   | [PR#105](https://github.com/BBC-News/psammead/pull/105) Create initial package, pulled in from [simorgh](https://github.com/BBC-News/simorgh).                         |
+| 0.1.1   | [PR#105](https://github.com/BBC-News/psammead/pull/202) Fixes a styling bug caused by an [incorrect constant import](https://github.com/BBC-News/psammead/issues/201). |
```
Comment (bcmn) [inline packages/components/psammead-brand/CHANGELOG.md]: Pedantic: You've only updated one of the PR numbers to `202`.
Even more pedantic: I think we normally do latest-first for changes?

Comment (pjlee11) [reply packages/components/psammead-brand/CHANGELOG.md]: Nice spot - Not pedantic but just upholding our high standards!

**Review by sareh — APPROVED** [review]

**Review by radiocontrolled — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM
